Zukini.UI.Examples.Features/App.config

```diff
@@ -1,37 +1,44 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <configSections>
-    <section name="specFlow" type="TechTalk.SpecFlow.Configuration.ConfigurationSectionHandler, TechTalk.SpecFlow"/>
+    <section name="specFlow" type="TechTalk.SpecFlow.Configuration.ConfigurationSectionHandler, TechTalk.SpecFlow" />
   </configSections>
-
   <appSettings>
     <!-- Test Execution Settings -->
-    <add key="Browser" value="Chrome"/>
-    <add key="Timeout" value="5"/>
-    <add key="RetryInterval" value=".1"/>
-
+    <add key="Browser" value="Chrome" />
+    <add key="Timeout" value="5" />
+    <add key="RetryInterval" value=".1" />
     <!-- Zukini Settings -->
-    <add key="MaximizeBrowser" value="true"/>
-    <add key="ScreenshotDirectory" value="..\..\..\Screenshots"/>
-
+    <add key="MaximizeBrowser" value="true" />
+    <add key="ScreenshotDirectory" value="..\..\..\Screenshots" />
     <!-- Application Under Test Settings -->
-    <add key="GoogleUrl" value="http://www.google.com"/>
-    <add key="W3SchoolsBaseUrl" value="http://www.w3schools.com"/>
-    <add key="JsonPlaceholderApiUrl" value="http://jsonplaceholder.typicode.com"/>
-
+    <add key="GoogleUrl" value="http://www.google.com" />
+    <add key="W3SchoolsBaseUrl" value="http://www.w3schools.com" />
+    <add key="JsonPlaceholderApiUrl" value="http://jsonplaceholder.typicode.com" />
     <!-- Grid Settings - Add Url to Grid Hub here -->
-    <add key="GridUrl" value=""/>
-
+    <add key="GridUrl" value="" />
+    <add key="ClientSettingsProvider.ServiceUri" value="" />
   </appSettings>
-
   <specFlow>
     <stepAssemblies>
-      <stepAssembly assembly="Zukini.UI"/>
-      <stepAssembly assembly="Zukini.UI.Examples.Features"/>
+      <stepAssembly assembly="Zukini.UI" />
+      <stepAssembly assembly="Zukini.UI.Examples.Features" />
     </stepAssemblies>
-    <unitTestProvider name="NUnit"/>
+    <unitTestProvider name="NUnit" />
   </specFlow>
   <startup>
-    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5.1"/>
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5.1" />
   </startup>
-</configuration>
+  <system.web>
```

Comment:
Did this come in with the added System.Web.Extensions? I would like to avoid adding a dependency unles necessary.